include/wx/cmndata.h

```diff
@@ -178,8 +178,14 @@ class WXDLLIMPEXP_CORE wxPrintDialogData: public wxObject
     wxPrintDialogData(const wxPrintData& printData);
     virtual ~wxPrintDialogData();
 
-    int GetFromPage() const { return m_printFromPage; }
-    int GetToPage() const { return m_printToPage; }
+    // Retained for backward compatibility.
+    // Returns the from page of the first range or 0 if there is no page range.
+    int GetFromPage() const;
+
+    // Retained for backward compatibility.
+    // Returns the to page of the first range or 0 if there is no page range.
```

Comment:
This is probably not very important in practice, but perhaps we should return `to` of the _last_ range here? So that if the user selects `1-2,8-9` in the dialog, an old application not updated to use the new API yet would print the pages `1-9` rather than `1-2`? Neither is ideal, of course, but the former is perhaps slightly better?

Reply:
Of course, we could, if it is better, you can decide..